File: NodeJs_TypeScript_JavaScript_Server/Project_NodeJs_VSCode/src/base/net/util/IpPort.ts
export class IpPort {

  public ip: string;
  public port: number;

  public constructor(ip: string, port: number) {
    this.set(ip, port);
  }
  
  public set(ip: string, port: number): void {
    this.ip = ip;
    this.port = port;
  }

}

File: CPP_Joc_Windows_Android/SH_Client_Win_Cpp_Cmake/App/src/base/net/util/IpPort.cpp
#include "IpPort.h"

using namespace base::Net;

IpPort::IpPort(std::string ip, int port)
    : ip(ip), port(port)
{
    //void
}

IpPort::IpPort() {
    //void
}

void IpPort::set(std::string& ip, int port) {
    this->ip = ip;
    this->port = port;
}

IpPort::~IpPort() {
    //void
}
